packages/identity-app/src/Identity.tsx

```diff
@@ -1,43 +1,45 @@
-// Copyright 2017-2018 @paritytech/substrate-light-ui authors & contributors
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { AddressSummary, Container, Input, MarginTop, NavButton, Stacked, WalletCard, WithSpace } from '@polkadot/ui-components';
+import keyring from '@polkadot/ui-keyring';
+
 import React from 'react';
-import { RouteComponentProps } from 'react-router';
-import { Container, Input, MarginTop, NavButton, Stacked, WalletCard, WithSpace } from '@polkadot/ui-components';
+import { RouteComponentProps } from 'react-router-dom';
 import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid';
 
-type Props = RouteComponentProps & {
-  basePath: string
-};
+interface Props extends RouteComponentProps {
+  basePath: string;
+}
 
 type State = {
-  seed?: string,
+  recoveryPhrase?: string,
   name?: string,
   lookupAddress?: string
 };
 
-export class Identity extends React.PureComponent<Props, State> {
+export class Identity extends React.Component<Props, State> {
   state: State = {};
 
-  private handleInputSeed = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>) => {
-    this.setState({ seed: value });
-  }
-
-  private handleInputName = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>) => {
-    this.setState({ name: value });
-  }
-
   private handleAddAccount = () => {
-    const { name, seed } = this.state;
+    const { name, recoveryPhrase } = this.state;
 
-    console.log(name, seed);
+    console.log(name, recoveryPhrase);
```

Comment:
(disregard this comment if you planned to remove the console.log, or add a FIXME so that we know we should implement something there.)

I would prefer to remove all console.logs from the final version. If there are useful logs for the user, we can use a package like `debug` to show those, so that user can filter the ones he wants to see.